js/bittrex.js

```diff
@@ -489,7 +489,11 @@ module.exports = class bittrex extends Exchange {
         let request = {};
         request[orderIdField] = id;
         let response = await this.marketGetCancel (this.extend (request, params));
-        return response;
+        return {
+            'info': response,
+            'id': id,
+            'status': 'canceled',
```

Comment:
Sorry, but we don't the status of the order at this point. So it's undefined until we fetch it. We cannot artificially set the status for `canceled` at this point. When the exchange accepts a request for canceling an order – that does not mean that the order is cancelled, a successful response merely means that the request for canceling was accepted.

Reply:
To clarify: we should only set this field if we know for sure the new status (if the exchange provides it in response to cancelOrder).

Reply:
You are absolutely right. Thank you for pointing it out. I have reverted that change in other pull requests, too. 

Also, see my proposal in #2983.